packages/icons/src/align-content.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const AlignContentStart = React.forwardRef<SVGSVGElement, IconProps>(
```

Comment:
maybe we should have a consistent icon per file structure, wdyt? the rest is like this

Reply:
I think they are in one category like many other icons would be, but they are separate icons